protocol/p2p/src/core/peer.rs

```diff
@@ -13,14 +13,16 @@ pub struct PeerProperties {
     pub time_offset: i64,
 }
 
-#[derive(Debug)]
+#[derive(Debug, Eq, PartialEq, Hash)]
 pub struct Peer {
     identity: PeerId,
     net_address: SocketAddr,
     is_outbound: bool,
     connection_started: Instant,
     properties: Arc<PeerProperties>,
     last_ping_duration: u64,
+    last_block_transfer: Option<Instant>, // TODO: add this to rpc, currently sidelined due to ongoing RPC development
```

Comment:
The "currently" part is redundant since it can change. Please only leave `// TODO: add this to rpc`

Reply:
done